Source/Core/DolphinQt/Config/Graphics/ColorCorrectionConfigWindow.cpp

```diff
@@ -157,24 +175,26 @@ void ColorCorrectionConfigWindow::ConnectWidgets()
     // For the moment we leave this enabled even when we are outputting in HDR
     // (which means they'd have no influence on the final image),
     // mostly because we don't have a simple way to determine if HDR is engaged from here
-    m_sdr_display_gamma_srgb->setEnabled(m_correct_gamma->isChecked());
+    m_sdr_display_target_srgb->setEnabled(m_correct_gamma->isChecked());
+    m_sdr_display_target_custom->setEnabled(m_correct_gamma->isChecked());
     m_sdr_display_custom_gamma->setEnabled(m_correct_gamma->isChecked() &&
-                                           !m_sdr_display_gamma_srgb->isChecked());
+                                           m_sdr_display_target_custom->isChecked());
   });
 
-  connect(m_sdr_display_gamma_srgb, &QCheckBox::toggled, this, [this] {
-    m_sdr_display_custom_gamma->setEnabled(m_correct_gamma->isChecked() &&
-                                           !m_sdr_display_gamma_srgb->isChecked());
+  connect(m_sdr_display_target_srgb, &QRadioButton::toggled, this, [this] {
+    Config::SetBaseOrCurrent(Config::GFX_CC_SDR_DISPLAY_GAMMA_SRGB,
+                             m_sdr_display_target_srgb->isChecked());
+    m_sdr_display_custom_gamma->setEnabled(!m_sdr_display_target_srgb->isChecked());
   });
 
   connect(m_sdr_display_custom_gamma, &ConfigFloatSlider::valueChanged, this, [this] {
     m_sdr_display_custom_gamma_value->setText(
-        QString::asprintf("%f", m_sdr_display_custom_gamma->GetValue()));
+        QString::asprintf("%.2f", m_sdr_display_custom_gamma->GetValue()));
```

Comment:
I think 3 points of precision would be more appropriate here

Reply:
By interacting with the slider, it's not possible to select values that would require a 3rd point of precision, hence why I capped it at two. You can still enter arbitrary values directly in the INI and they'll be honored, but since two points of precision is the maximum granularity you can get from the UI controls, I don't think we should expose more points...